monitoring/linear_combination.py

```diff
@@ -0,0 +1,94 @@
+from operator import itemgetter
+
+import iris
+from scriptengine.exceptions import (
+    ScriptEngineTaskArgumentInvalidError,
+    ScriptEngineTaskArgumentMissingError,
+    ScriptEngineTaskRunError,
+)
+from scriptengine.tasks.core import Task, timed_runner
+
+import helpers.cubes
+
+
+class LinearCombination(Task):
+
+    _required_arguments = (
+        "src",
+        "dst",
+    )
+
+    def __init__(self, arguments):
+        LinearCombination.check_arguments(arguments)
+        super().__init__(arguments)
+
+    @timed_runner
+    def run(self, context):
+        self.log_info("Create new netCDF file from linear combination of inputs")
+
+        sources = self.getarg("src", context)
+        if not isinstance(sources, list):
+            self.log_error("Invalid 'src' argument, must be a list")
+            raise ScriptEngineTaskArgumentInvalidError
+
+        result = 0
```

Comment:
You chose 0 here to be able to add cubes to it? A nice way to fix this issue, although it seems a bit weird that we initially assign an `int` to result but later on access variables like `result.var_name`. Maybe a comment would be nice here?

Reply:
Don't know. I my thinking, `result` is the (scaled) sum of cubes and this sum is initialised with 0. Not an integer, but a conceptional zero. Iris is thankfully taking care of the details of adding/multiplying cubes and numbers and I am taking advantage of it. But maybe a comment to clarify doesn't hurt.